gradle.properties

```diff
@@ -37,10 +53,11 @@ javaxServletVersion=3.1.0
 javaxElVersion=3.0.0
 javaxElImplVersion=2.2.6
 javaxValidationVersion=1.1.0.Final
-javassistVersion=3.21.0-GA
+javassistVersion=3.22.0-GA
 javaxJstlVersion=1.2
 
-semverVersion=2.0.3
+semverVersion=2.1.0
```

Comment:
I am not quite if all of these versions are in fact needed, and/or the dependencies. Very likely, you should try to remove as many of them as possible.